client/components/events/EventViewToggle.tsx

```diff
@@ -79,8 +70,6 @@ export const EventViewToggle = ({ events }: EventViewProps) => {
 
   return (
     <EventViewToggleContainer>
-      <EventViewToggleButtonsContainer>
-        <EventViewToggleButtonsSpacer />
         <EventViewToggleButtons>
```

Comment:
indentation is kind of broken here, can we re-run Prettier on this?

Reply:
Done